conans/client/cmd/export.py

```diff
@@ -208,11 +208,16 @@ def _replace_scm_data_in_conanfile(conanfile_path, scm_data):
     tree = ast.parse(content)
     to_replace = []
     comments = []
+    class_line = None
+    tab_size = 4
```

Comment:
🤞 I've been having a look to `ast` module and I can't find a way to detect the tab size or the tabs-vs-spaces indentation... If this bites us, then it will be the time to reimplement all this function and dump the modified AST instead of doing string replacements

Reply:
yeah, I think the same way, if this holds, ok, otherwise, we have to do it better.